Sources/MongoSwift/MongoClient.swift

```diff
@@ -241,25 +226,65 @@ public class MongoClient {
     }
 
     /**
-     * Get a list of databases.
+     * Run the `listDatabases` command.
      *
      * - Parameters:
-     *   - options: Optional `ListDatabasesOptions` to use when executing the command
+     *   - filter: Optional `Document` specifying a filter that the listed databases must pass. This filter can be based
+     *     on the "name", "sizeOnDisk", "empty", or "shards" fields of the output.
      *
      * - Returns: A `MongoCursor` over `Document`s describing the databases matching provided criteria
      *
      * - Throws:
-     *   - `UserError.invalidArgumentError` if the options passed are an invalid combination.
      *   - `UserError.logicError` if the provided session is inactive.
      *   - `EncodingError` if an error is encountered while encoding the options to BSON.
+     *
+     * - SeeAlso: https://docs.mongodb.com/manual/reference/command/listDatabases/
+     */
+    public func listDatabases(_ filter: Document? = nil,
```

Comment:
I think its worth noting/discussing what we're gaining and giving up with this approach. Not using the options struct might feel more ergonomic, however it does move us towards a few potential risks:
  - It makes extension of these methods more difficult. What if I wanted to specify a client-side timeout (`maxTimeMS`) for this operation? Should it be added as an additional parameter?
  - Lets say we just add it as a parameter, what do we do with the old method? Mark it deprecated, and delete it sometime in the future in a major version release?
  - It definitely expands the API surface we need to maintain

Its a bit weird to bring this up in a PR _about_ adding a helper, but one of the drivers mantras is to [Minimize Administrative Helpers](https://github.com/mongodb/specifications#minimize-administrative-helpers). It could be argued that adding three methods, when one would suffice, goes against this principle. 

Anyway, I don't feel _very_ strongly about this, but I think its worth mentioning.

Reply:
> Lets say we just add it as a parameter, what do we do with the old method?
I don't think we'd need a new method, it would just be an additional optional parameter that defaults to `nil` and all existing calls to it would keep working 

that said I feel like that kind of stuff goes in an options struct, so we would end up needing a public options struct again 🤔

Reply:
FWIW note that `listCollections` also sticks `filter` into an options struct so if we don't have a ticket about making that line up with this stuff we should make one

Reply:
We've debated this thoroughly in person :) I'm going to resolve it now.